tests/common.cpp

```diff
@@ -514,22 +592,6 @@ QString randomMessage(int words)
     return msg;
 }
 
-bool waitSignal(QSignalSpy &spy, int msec)
```

Comment:
A couple of things: there's no QVERIFY inside this method, so the commit message isn't precisely correct.  Also, this one has a wait-for-msec parameter, whereas QTRY_COMPARE will use some predefined wait timeout, I guess.  This last one shouldn't be a problem, but maybe it's worth mentioning in the commit message that the custom timeout parameter isn't required any more, so QTRY_COMPARE suffices.

Reply:
Yes true, fixed.